upflux-client/src/features/Navbar/Navbar.tsx

```diff
@@ -1,19 +1,37 @@
+import React from 'react';
 import { Container, Image } from '@mantine/core';
-import { Link } from 'react-router-dom'; 
+import { Link } from 'react-router-dom';
+import notifBell from "../../assets/images/notif_bell.png";
 import logo from "../../assets/logos/logo-no-name.png";
 import './navbar.css';
 
-export const Navbar = () => {
+// Accept a prop `home` to conditionally render the navigation items
+interface NavbarProps {
+  home: boolean;
+}
+
+export const Navbar: React.FC<NavbarProps> = ({ home }) => {
```

Comment:
` export const Navbar: React.FC<NavbarProps> = ({ home }) =>` Change to 

```
export const Navbar: React.FC<NavbarProps> = (props : NavbarProps) => {
 const { home } = props;

}
```

also change home name to something else as mentioned above